### DISCUSSION TITLE
Implement skin editor layer that can be captured by OBS only

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#27897] [Ideas]
Would be nice having an ability in the future to place skin elements on the screen only for viewers at the stream (twitch, youtube, etc.).

**Comment by SupDos** [comment]
I'd be curious to know if this is even technically possible, but I think your best bet would be to wait for websockets to be added (https://github.com/ppy/osu/pull/18129), and then integrate that as an OBS source in whichever way you want.

**Reply by Loreos7** [reply]
I think bdach could have answered that...

**Comment by peppy** [comment]
Would require a separate window. Unless someone does this for us it's somewhere between way-in-the-future to nope.
